### ISSUE TITLE
Day reminder (# mon, tue, wed, thu, fri, sat, sun) not working

### ISSUE THREAD

**Issue by iu2kul** [UbhiTS/ad-alexatalkingclock#21]
Hi there,
after some time to understand how to setup this app I succeded to keep it running.

All is fine about #daily advise; every hour Alexa gives hour annuncment.

They day reminder, # mon, tue, wed, thu, fri, sat, sun, is not working.

Note: to keep this app running I copy che .py file in this root /addon_configs/a0d7b954_appdaemon/apps, after installing AppDaemon otherwise nothing is working. On app.yaml I add the config file as requested.

Where I was wrong ?

Thanks in advance for any support

Franco

**Comment by iu2kul** [comment]
Hi there, me again. The day reminder is working but only when you setup the time programmed in the system. I mean if you setup the advise every hour so you can have the reminder eache hour. The only point is the date and day is in English and not in local language

**Comment by UbhiTS** [comment]
diagnosing the issue now. My apologies, I wasn't looking at my github notifications

**Comment by UbhiTS** [comment]
yeah, that's the current constraint, the reminders will only override the message spoken by alexa at the time specified. I've updated the docs a bit to clarify. 

      # IMPORTANT NOTE: ensure you set the reminder times to trigger at the announcement times only. See below for more details. 
      #   if you set a reminder at 08:15:00, or 08:45:00 it will only trigger if the announcements > quarter_hour is set to true, else it will get ignored.
      #   if you set a reminder at 08:30:00, it will only trigger if the announcements > half_hour is set to true, else it will get ignored.

**Comment by UbhiTS** [comment]
also, I pull the day name from the python calendar library, i'll spend some time later in a couple of weeks to learn how to internationalize it, what language are you looking for?
